src/Compilers/Core/RebuildTest/CSharpRebuildTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using System.Reflection.PortableExecutable;
+using BuildValidator;
+using Castle.Core.Logging;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Emit;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.Extensions.Logging;
+using Xunit;
+
+namespace Rebuild.UnitTests
+{
+    //[CompilerTrait(CompilerFeature.)]
+    public class CSharpRebuildTests : CSharpTestBase
```

Comment:
What about `VB` tests here?

Reply:
I think it would be good to have a VB version of this test. I guess in that case we'd want to add a reference to the VB test helpers?

Reply:
My instinct was to have an `abstract` base class and then have both a VB and C# derivation. Hoping that it would be easy to abstract over the languages this way.

Reply:
I'd like to introduce a few more test cases and then try to figure out what test helpers would be most useful.